main.tf

```diff
@@ -354,6 +361,9 @@ resource "google_compute_forwarding_rule" "internal_load_balancer" {
   network               = "${google_compute_network.protected_vpc_network.self_link}"
   subnetwork            = "${google_compute_subnetwork.protected_subnet.self_link}"
 }
+resource "google_compute_address" "master_external_address" {
+  name = "${var.cluster_name}-external-address-${random_string.random_name_post.result}"
+}
```

Comment:
I am just curious if there's resource name length limit in your resources?

Reply:
If you attempt to put one that is too large Terraform will generate an error like so :

```
Error: "name" ("fortigateautoscale-externaaslkdalsdkjflasdjflasjdflaklasdhjfalskdhfjlaksdhfkalsdhfklasdhfaksldhfaksldhfksdjhfskdjhfaksjdhfkashdfkhfaskldhfasdfl-address-toxuy") doesn't match regexp "^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$"
```